modules/ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries.adoc

```diff
@@ -0,0 +1,404 @@
+:_content-type: PROCEDURE
+[id="ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries_{context}"]
+= Configuring the hub cluster to use unauthenticated registries
+:context: ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries
```

Comment:
Lose this! Only set `:context:` in assemblies :)

Reply:
Updated as suggested.